src/main/com/fulcrologic/rad/report_options.cljc

```diff
@@ -569,3 +569,16 @@
   "Report option. Should the page number and other controls be tracked in the URL? Defaults to TRUE. Useful if embedding
    a report into someething like a modal, where the URL parameters are not meant to affect or be modified by the content."
   :com.fulcrologic.rad.report/track-in-url?)
+
+(def initialize-ui-props
+  "Report-only option, meant for setting up additinal component state (particularly `:ui/???` props that were added
+   in a query inclusion) when the report is created or loaded.
+
+   Either a map or a `(fn [ReportClass props] props)` that, when present, is called when an instance of this
```

Comment:
Initial state has params, not props.  If I called it props in forms it may be I'm doing it differently there, but in any case this docstring should be fixed to reflect reality.